script/tasks/diamondSyncSigs.sh

```diff
@@ -94,10 +94,10 @@ function diamondSyncSigs {
       # call diamond
       if [[ "$DEBUG" == *"true"* ]]; then
         # print output to console
-        cast send "$DIAMOND_ADDRESS" "batchSetFunctionApprovalBySignature(bytes4[],bool)" "[${PARAMS::${#PARAMS}-1}]" true --rpc-url $RPC_URL --private-key $(getPrivateKey "$NETWORK" "$ENVIRONMENT") --legacy
+        cast send "$DIAMOND_ADDRESS" "batchSetFunctionApprovalBySignature(bytes4[],bool)" "[${PARAMS::${#PARAMS}-1}]" true --rpc-url $RPC_URL --account $(getAccount "$NETWORK" "$ENVIRONMENT") --legacy
```

Comment:
no --password flag needed?

same applies for all other cast send calls in this file